nodes/widgets/ui_dropdown.js

```diff
@@ -20,6 +20,10 @@ module.exports = function (RED) {
                     // dynamically set "label" property
                     statestore.set(group.getBase(), node, msg, 'label', msg.label)
                 }
+                if (msg.multiple !== undefined) {
```

Comment:
```suggestion
                if (typeof msg.multiple !== 'undefined') {
```

typeof x !== 'undefined' is a more robust and safe way to check for the existence of a variable